internal/release/dynatrace-otel-collector.conf

```diff
@@ -0,0 +1,5 @@
+# Systemd environment file for the otelcol service
+
+# Command-line options for the otelcol service.
+# Run `/usr/bin/dynatrace-otel-collector --help` to see all available options.
+OTELCOL_OPTIONS="--config=/etc/dynatrace-otel-collector/config.yaml"
```

Comment:
Will this one only be set in context of this process or globally? If globally, it would clash with a potential upstream collector service.

Reply:
those variables are only in the environment of the service, not global